app/js/ContentCacheMetrics.js

```diff
@@ -20,6 +36,16 @@ function emitPdfCachingStats(stats, timings) {
       ? timings.compileE2E /
         (timings.compileE2E - timings['compute-pdf-caching'])
       : 1
+  if (fraction > 1.5) {
```

Comment:
How about a limit to ignore short compiles as well? `if (fraction > 1.5 && timings.compileE2E > 10*1000)`

Reply:
Done in https://github.com/overleaf/clsi/pull/253/commits/c2e72151931b1c9ee8c5959216c10ad5a4065f69